testfiles/API/user_defined_equipment.py

```diff
@@ -0,0 +1,480 @@
+# EnergyPlus, Copyright (c) 1996-2024, The Board of Trustees of the University
```

Comment:
This file is just PythonPluginUserDefinedWindowAC or however it's named, but instead of a PythonPlugin which gets called from EnergyPlus, it's a standalone Python function that calls out to EnergyPlus.